scipy/stats/_qmc.py

```diff
@@ -950,18 +968,56 @@ class LatinHypercube(QMCEngine):
            [6.80338249, 3.08795949],
            [2.65448791, 3.83491828]])
 
+    Alternatively, optimized LHS can be constructed with `optimization`.
+    It should give a sample with a lower discrepancy at a higher computational
```

Comment:
"should" is unnecessary here, IMO. Or, this can all be:

> Use the `optimized` keyword argument to produce an LHS with
lower discrepancy at higher computational cost.